ui/components/MesheryFilters/FiltersCard.js

```diff
@@ -112,16 +114,17 @@ function FiltersCard({
               >
                 <DoneAllIcon className={classes.iconPatt}/>
               Deploy
-              </Button>
-              {visibility === "public" ? <Button
+              </TooltipButton>
+              {visibility === "public" ? <TooltipButton
+                title="clone"
```

Comment:
Please be consistent with capitalization.

Reply:
@leecalcote sorry didn't understand what you mean by  consistent with capitalization. Do you mean  TooltipButton should be named something else?

Reply:
No, the text contained within. e.g. Clone

Reply:
Got it